next.config.js

```diff
@@ -44,6 +59,17 @@ const nextConfig = {
             type: 'asset/source'
         });
 
+        const useFallbackForInternalPackages = process.env.USE_FALLBACK_FOR_INTERNAL_PACKAGES === '1';
+
+        config.resolve.alias = config.resolve.alias || {};
+
+        if (useFallbackForInternalPackages) {
+            config.resolve.alias['@webteam'] = path.resolve(
+                __dirname,
+                'components/webteam-fallback'
+            );
+        }
```

Comment:
this is the place, where all the magic happens.
We just resolve another module if it is an external user.